android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -1442,16 +1427,14 @@ public void onClick(View v) {
                 }
 
                 switch (status) {
-                    case BraveRewardsExternalWallet.NOT_CONNECTED:
-                    case BraveRewardsExternalWallet.CONNECTED:
-                    case BraveRewardsExternalWallet.PENDING:
-                    case BraveRewardsExternalWallet.VERIFIED:
-                        if (status == BraveRewardsExternalWallet.NOT_CONNECTED) {
+                    case WalletStatus.kNotConnected:
+                    case WalletStatus.kConnected:
+                        if (status == WalletStatus.kNotConnected) {
                             TabUtils.openUrlInNewTab(false,
                                     BraveActivity.BRAVE_REWARDS_SETTINGS_WALLET_VERIFICATION_URL);
                             dismiss();
                         } else {
-                            int requestCode = (status == BraveRewardsExternalWallet.NOT_CONNECTED)
+                            int requestCode = (status == WalletStatus.kNotConnected)
```

Comment:
The `if``'s condition above is the same, so this should likely be `int requestCode = BraveConstants.USER_WALLET_ACTIVITY_REQUEST_CODE;`.